packages/chain-adapters/src/evm/polygon/PolygonChainAdapter.ts

```diff
@@ -0,0 +1,96 @@
+import type { AssetId } from '@shapeshiftoss/caip'
+import { ASSET_REFERENCE, polygonAssetId } from '@shapeshiftoss/caip'
+import type { BIP44Params } from '@shapeshiftoss/types'
+import { KnownChainIds } from '@shapeshiftoss/types'
+import * as unchained from '@shapeshiftoss/unchained-client'
+
+import type { FeeDataEstimate, GetFeeDataInput } from '../../types'
+import { ChainAdapterDisplayName } from '../../types'
+import { bn, bnOrZero, calcFee } from '../../utils'
+import type { ChainAdapterArgs } from '../EvmBaseAdapter'
+import { EvmBaseAdapter } from '../EvmBaseAdapter'
+import type { GasFeeDataEstimate } from '../types'
+
+const SUPPORTED_CHAIN_IDS = [KnownChainIds.PolygonMainnet]
+const DEFAULT_CHAIN_ID = KnownChainIds.PolygonMainnet
+
+export class ChainAdapter extends EvmBaseAdapter<KnownChainIds.PolygonMainnet> {
```

Comment:
`packages/chain-adapters/src/utils/index.ts` needs update as well was only missing file I see in `chain-adapters`